Swamp Sneak/Assets/Scripts/Classes/Player.cs

```diff
@@ -2,7 +2,64 @@
 using System.Collections.Generic;
 using UnityEngine;
 
+
 public class Player : Person {
+	private GameControl control;
```

Comment:
This property can actually be moved up to the Entity class with a `protected` access modifier. That way all of the children classes will have reference to the GameControl without having to explicitly create a variable in each of them.

Reply:
Good idea!